addons/survey/static/src/js/survey_result.js

```diff
@@ -154,55 +218,46 @@ if(!$('.js_surveyresult').length) {
                         return D3_COLORS[index % 20];
                     }),
                 }]
-        }
+            }
         };
         return chartConfig;
-    }
+    },
 
     //load chart to svg element chart:initialized chart, response:AJAX response, quistion_id:if of survey question, tick_limit:text length limit
-    function load_chart(chartConfig, containerSelector){
+    _load_chart: function (chartConfig, containerSelector) {
         var $container = $(containerSelector).css({position: 'relative'});
         var $canvas = $container.find('canvas');
         var ctx = $canvas.get(0).getContext('2d');
         return new Chart(ctx, chartConfig);
-    }
-
-    //Script For Graph
-    var survey_graphs = $('.survey_graph');
-    $.each(survey_graphs, function(index, graph){
-        var question_id = $(graph).attr("data-question_id");
-        var graph_type = $(graph).attr("data-graph_type");
-        var graph_data = JSON.parse($(graph).attr("graph-data"));
-        var containerSelector = '#graph_question_' + question_id;
-        var chartConfig;
-        if(graph_type == 'multi_bar'){
-            chartConfig = init_multibar_chart(graph_data);
-            return load_chart(chartConfig, containerSelector);
-        }
-        else if(graph_type == 'bar'){
-            chartConfig = init_bar_chart(graph_data);
-            return load_chart(chartConfig, containerSelector);
+    },
+
+    // -------------------------------------------------------------------------
+    // Handlers
+    // -------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onSurveyAnswerMouseHover: function (ev) {
```

Comment:
Maybe try to set the filter as `.filter { display: none; }` and add a class `td:hover { .filter { display: block; } }`
To try to solve this with plain css instead.